velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -1764,6 +1763,8 @@ TEST_F(MinMaxByNTest, sortedGroupBy) {
 }
 
 TEST_F(MinMaxByNTest, variableN) {
+  AggregationTestBase::disableTestIncremental();
```

Comment:
can you please remind me what this does and why we moved it here

Reply:
testIncremental() is used for testing with valid inputs and testIncremental internally uses global aggregation. The tests under variableN are invalid inputs with global aggregation, so I disable it in this unit test.